diff --git a/node_modules/duckduckgo-images-api/src/constants.js b/node_modules/duckduckgo-images-api/src/constants.js
index 66f70e1..26e93d0 100644
--- a/node_modules/duckduckgo-images-api/src/constants.js
+++ b/node_modules/duckduckgo-images-api/src/constants.js
@@ -2,7 +2,7 @@ module.exports = {
     url : 'https://duckduckgo.com/',
     headers : {
         'dnt': '1',
-        'accept-encoding': 'gzip, deflate, sdch',
+        'accept-encoding': 'gzip, deflate',
         'x-requested-with': 'XMLHttpRequest',
         'accept-language': 'en-GB,en-US;q=0.8,en;q=0.6,ms;q=0.4',
         'user-agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36',
